src/awsexplorer/defaultRegion.ts

```diff
@@ -113,31 +113,31 @@ export async function checkExplorerForDefaultRegion(
 
     const regionHiddenResponse = response[0].label
 
-    switch (regionHiddenResponse) {
```

Comment:
why disallow switch-case?

Reply:
declaring variables that escape the switch case. In this case though, I thought the logic was very strange, I think it is more clear now.